### PR TITLE
A module that will retrieve and parse security.txt files.

### PR DESCRIPTION
**Pull request by steve-bate**
#514

### CODE REVIEW

**Comment by steve-bate** [comment]
The text fixtures/stubs that I used to unit test the module could be pulled into a test-support library (and extended) to support testing other modules.

File: modules/sfp_securitytxt.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:        sfp_securitytxt
+# Purpose:     Find and parse .well-known/security.txt files.
+#
+# Author:      Steve Micallef <steve@binarypool.com>
```
Comment (smicallef) [inline modules/sfp_securitytxt.py:6]: This should reflect you as the author :)

**Comment by smicallef** [comment]
Style-wise there are many differences to other modules. Could you refactor the module to fit with the others? I know the style is not ideal, but I'd rather they all be consistent and then think of an ideal new structure as a separate effort.

File: modules/sfp_securitytxt.py (hunk 1 above)
Comment (steve-bate) [reply modules/sfp_securitytxt.py:6]: I originally had my name there, but I wasn't sure how to handle the copyright. Should I just replace the name and leave you as the copyright holder?

**Comment by steve-bate** [comment]
Can you give me a few examples about stylistic differences that would concern you?

File: modules/sfp_securitytxt.py (hunk 1 above)
Comment (smicallef) [reply modules/sfp_securitytxt.py:6]: Your name there as the author is fine.

**Comment by smicallef** [comment]
Take a look at other modules, specifically `setup()` and note there is no `__init__()` in the other modules. Also naming like `self.sf` instead of `self.spiderfoot` and others.

**Comment by steve-bate** [comment]
OK, thanks for the clarification. I added the __init__ because of some static analysis warnings (about creating instance fields outside of __init__), but I can remove it and disable the analysis warnings. I didn't realize the spiderfoot vs sf naming would be an issue since the field is private to this module and I prefer to use expressive variable names. I'm guessing that you maybe didn't like the names of some of the other private fields and function parameters too.

**Comment by smicallef** [comment]
Keep in mind that SpiderFoot has evolved from around 2012 when I didn't know Python at all, so there are a lot of relics in the code that could be re-written. At this stage however I'd prefer that the modules are consistent so that eventually it will be a simpler exercise to convert them to a new structure en masse.

**Comment by steve-bate** [comment]
I understand. I'll pull the PR and resubmit it after I rewrite it.

**Pull request closed without merging**